packages/react-scripts/template/README.md

```diff
@@ -1218,15 +1215,11 @@ Learn more about React Storybook:
 
 ## Making a Progressive Web App
 
-You can turn your React app into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) by following the steps in [this repository](https://github.com/jeffposnick/create-react-pwa).
-
-<!--
-TODO: uncomment
 By default, the production build is a fully functional, offline-first
 [Progressive Web App](https://developers.google.com/web/progressive-web-apps/).
 
```

Comment:
We need an intro paragraph selling user on PWAs here. Can grab something from Google docs and rephrase.